MapboxNavigation/NavigationMapView.swift

```diff
@@ -433,6 +433,12 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         return candidates
     }
     
+    var userPuckScreenCoordinates: CLLocationCoordinate2D? {
+        guard let rect = userCourseView?.frame else { return nil }
+        let point = CGPoint(x: rect.midX, y: rect.midY)
+        return self.convert(point, toCoordinateFrom: self)
```

Comment:
Is there a better method we can use here that accounts for the whole frame instead of a point?

Reply:
No; the only other option is to convert a rect, but that won’t account for rotation or pitch.